packages/typespec-azure-resource-manager/lib/foundations/arm.foundations.tsp

```diff
@@ -44,6 +44,17 @@ model DefaultBaseParameters<Resource extends {}> {
   ...ResourceUriParameter;
 }
 
+/**
+ * Standard type definition for Azure Resource Manager Tags property.
+ *
+ * It is included in the TrackedResource template definition.
+ */
+/** The Azure Resource Manager Resource tags. */
+model ArmTagsProperty {
```

Comment:
Isn't this a breaking change?  People do add tags to their proxy resources.  Seems like we should have a linting rule for this rather than move the model

Reply:
No. ArmTagsProperty was originally in `types.tsp` under common-types folder BUT the namespace was `Foundations`. As I moved everything in types.tsp to under `CommonTypes` namespace, moving this back into foundations.tsp will keep it unchanged. In fact, we may want to consider moving/aliasing it into ARM namespace so that you can directly spread it and limit dependency on Foundations namespace.